### ISSUE TITLE
Login.tsx screen email field missing accessibility label

### ISSUE THREAD

**Issue by dshiverware** [brightlayer-ui/react-native-workflows#200]
On the `Login.tsx` screen, the `password-text-field` appears to have both a testID and an accessbilityLabel but the `email-text-field` only has a testID. This is causing a failure in our Appium test suite. Can the accesbilityLabel `accesbilityLabel ={'email-text-field'}`be added to the email field?

**Comment by dshiverware** [comment]
Hi @daileytj, is there an ETA for the 5.1.0 release?

**Comment by daileytj** [comment]
> Hi @daileytj, is there an ETA for the 5.1.0 release?

I'll be going through our backlog/bugs tomorrow to structure our milestones, but I can publish a beta package with the test-id for now if that will help.

**Comment by dshiverware** [comment]
That would be great, thanks!
